dune-project

```diff
@@ -164,6 +164,16 @@
  (conflicts
   (result (< 1.5))))
 
+; Dune plugin
+(package
+ (name ortac-dune)
+ (synopsis "Generate dune rules for other ortac plugins")
+ (description "Generate dune rules for other ortac plugins")
+ (authors
+  "Nicolas Osborne <nicolas.osborne@tarides.com>")
+ (maintainers "Nicolas Osborne <nicolas.osborne@tarides.com>")
+ (depends (ocaml (>= 4.11.0)) fmt (cmdliner (>= 1.1.0)) ortac-core (ortac-qcheck-stm :with-test)))
```

Comment:
We don’t use strict formatting in `dune-project` but I notice we tend to write each dependency on its own line in the other packages. We might keep the style.